tests/integration/test_remoted/test_agent_communication/test_shared_configuration.py

```diff
@@ -17,7 +17,7 @@
 # Configuration
 test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
 configurations_path = os.path.join(test_data_path, 'wazuh_shared_configuration.yaml')
-agent_conf_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data', 'agent.conf')
+agent_conf_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data', 'shared.conf')
```

Comment:
This file doesn't exist in the repository. It refers to this [file](https://github.com/wazuh/wazuh-qa/blob/9f9efc37dbceb2f9066f2d55528c5412498b708a/deps/wazuh_testing/wazuh_testing/data/agent.conf) which is used to create a new remote configuration file and push it to the agent. If so, you will need to rename the file